gammapy/image/maps.py

```diff
@@ -260,8 +260,8 @@ def solid_angle(self):
         """
         Solid angle image
         """
-        xsky, ysky = self.coordinates(mode='edges')
-        omega = -np.diff(xsky, axis=1)[1:, :] * np.diff(ysky, axis=0)[:, 1:]
+        coord = self.coordinates(mode='edges')
+        omega = -np.diff(coord.l, axis=1)[1:, :] * np.diff(coord.b, axis=0)[:, 1:]
```

Comment:
I think this will fail if the `wcs` is not in specified in galactic coordinates. It would be better to use `coordinates(coord_type='galactic', mode='edges')` in `solid_angle()` which behaves the "old way". Could you add a minimal test to `gammapy/image/tests/test_maps.py`? Thanks!